### ISSUE TITLE
Style guide?

### ISSUE THREAD

**Issue by reecebisel** [superposition-dev/situs#4]
Is it worth building out a style guide out? I was thinking of waiting until its really needed. Thoughts?

**Comment by lollar** [comment]
Yeah I think that waiting makes sense. We may want one for our design standards when we have them.

**Comment by reecebisel** [comment]
👍
